contributing.md

```diff
@@ -1,5 +1,22 @@
 Contributing
 ===
+Welcome to the world of content development in High Fidelity! This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).
```

Comment:
Welcome to the world of content development in High Fidelity!

This project is actively maintained by the High Fidelity Experiences development team and we welcome contributions of content that you'd like to share with the community under the [Apache 2.0 License](https://github.com/highfidelity/hifi-content/blob/master/LICENSE), either as part of your own work or through a request **for compensated work** made as part of the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc).

OR 

either as part of your own work or **through the [High Fidelity Development Fund](https://blog.highfidelity.com/cashing-out-earn-and-exchange-high-fidelity-coins-hfc-for-u-s-dollars-bfa3b8cb3ebc) where developers are compensated for their work.**

Added: Spacing and "for compensated work"
I think highlighting compensation is desireable. If that is not our intention.